doc/changes/latest.inc

```diff
@@ -73,6 +73,8 @@ Bugs
 
 - Fix bug in :func:`mne.viz.plot_alignment` where ECoG and sEEG channels were not plotted and fNIRS channels were always plotted in the head coordinate frame by `Eric Larson`_ (:gh:`8393`)
 
+- When reading BrainVision raw data, the channel units and types were sometimes not inferred correctly, by `Richard Höchenberger`_ (:gh:`8434`)
```

Comment:
I think we don't put a comma before "by".

Reply:
I've seen mixed use here… personally I prefer the "with comma" approach, but currently it's inconsistent, yes. I can remove the comma.